pulp_container/app/registry_api.py

```diff
@@ -1084,7 +1084,7 @@ def handle_safe_method(self, request, path, pk):
                 else:
                     raise ManifestNotFound(reference=pk)
 
-            return redirects.issue_manifest_redirect(manifest)
+            return Response(manifest.data)
```

Comment:
Bear in mind that we would like to return headers as well, similarly to `return web.Response(text=tag.tagged_manifest.data, headers=response_headers)`.

Reply:
@lubosmj What headers would you like to return in this case?